pkgs/top-level/all-packages.nix

```diff
@@ -9277,6 +9277,8 @@ in
   mozart2-binary = callPackage ../development/compilers/mozart/binary.nix { };
 
   nim = callPackage ../development/compilers/nim { };
+  nim-unwrapped = nim.unwrapped;
```

Comment:
Do these need to be listed at top-level?

Reply:
I listed those at the top-level so that `callPackage` feeds them back to the nim-wrapper. I did this because I want to be able to patch the stdlib from an overaly without rebuilding the compiler, just the wrapper. Not sure this is the best way to do this.